Index: order-service/order-domain/order-application-service/src/main/java/com/online/medicine/application/order/service/domain/ports/input/message/listener/payment/PaymentResponseMessageListener.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/order-service/order-domain/order-application-service/src/main/java/com/online/medicine/application/order/service/domain/ports/input/message/listener/payment/PaymentResponseMessageListener.java b/order-service/order-domain/order-application-service/src/main/java/com/online/medicine/application/order/service/domain/ports/input/message/listener/payment/PaymentResponseMessageListener.java
--- a/order-service/order-domain/order-application-service/src/main/java/com/online/medicine/application/order/service/domain/ports/input/message/listener/payment/PaymentResponseMessageListener.java	(revision ea6591da97963fad13718bfffd43fb4266e151ff)
+++ b/order-service/order-domain/order-application-service/src/main/java/com/online/medicine/application/order/service/domain/ports/input/message/listener/payment/PaymentResponseMessageListener.java	(date 1727692764416)
@@ -1,0 +1,16 @@
+package com.online.medicine.application.order.service.domain.ports.input.message.listener.payment;
+
+import com.online.medicine.application.order.service.domain.dto.messaging.PaymentResponse;
+
+public interface PaymentResponseMessageListener {
+    void paymentCompleted(PaymentResponse paymentResponse);
+    void paymentCancelled(PaymentResponse paymentResponse);
+}
+package com.online.medicine.application.order.service.domain.ports.input.message.listener.payment;
+
+import com.online.medicine.application.order.service.domain.dto.messaging.PaymentResponse;
+
+public interface PaymentResponseMessageListener {
+    void paymentCompleted(PaymentResponse paymentResponse);
+    void paymentCancelled(PaymentResponse paymentResponse);
+}
Index: order-service/order-domain/order-application-service/src/main/java/com/online/medicine/application/order/service/domain/ports/input/message/listener/pharmacyapproval/PharmacyApprovalMessageListener.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/order-service/order-domain/order-application-service/src/main/java/com/online/medicine/application/order/service/domain/ports/input/message/listener/pharmacyapproval/PharmacyApprovalMessageListener.java b/order-service/order-domain/order-application-service/src/main/java/com/online/medicine/application/order/service/domain/ports/input/message/listener/pharmacyapproval/PharmacyApprovalMessageListener.java
--- a/order-service/order-domain/order-application-service/src/main/java/com/online/medicine/application/order/service/domain/ports/input/message/listener/pharmacyapproval/PharmacyApprovalMessageListener.java	(revision ea6591da97963fad13718bfffd43fb4266e151ff)
+++ b/order-service/order-domain/order-application-service/src/main/java/com/online/medicine/application/order/service/domain/ports/input/message/listener/pharmacyapproval/PharmacyApprovalMessageListener.java	(date 1727692764426)
@@ -1,0 +1,16 @@
+package com.online.medicine.application.order.service.domain.ports.input.message.listener.pharmacyapproval;
+
+import com.online.medicine.application.order.service.domain.dto.messaging.PharmacyApprovalResponse;
+
+public interface PharmacyApprovalMessageListener {
+    void orderApproved(PharmacyApprovalResponse pharmacyApprovalResponse);
+    void orderRejected(PharmacyApprovalResponse pharmacyApprovalResponse);
+}
+package com.online.medicine.application.order.service.domain.ports.input.message.listener.pharmacyapproval;
+
+import com.online.medicine.application.order.service.domain.dto.messaging.PharmacyApprovalResponse;
+
+public interface PharmacyApprovalMessageListener {
+    void orderApproved(PharmacyApprovalResponse pharmacyApprovalResponse);
+    void orderRejected(PharmacyApprovalResponse pharmacyApprovalResponse);
+}
